policy/modules/system/init.fc

```diff
@@ -7,6 +7,8 @@
 /etc/rc\.d/init\.d/.*	--	gen_context(system_u:object_r:initrc_exec_t,s0)
 /etc/sysconfig/network-scripts/ifup-ipsec -- gen_context(system_u:object_r:initrc_exec_t,s0)
 
+/etc/systemd/system(/.*)?	gen_context(system_u:object_r:systemd_unit_t,s0)
```

Comment:
This should already be handled by the `file_contexts.subs_dist`.

Reply:
You are right. I think this is just a simple oversight.